day2.py

```diff
@@ -0,0 +1,32 @@
+
+def helper(vals):
+    # Checks all rules
+    # Check if list is ascending/descending and also if values differ between values 1-3
+    return (all(vals[i]<vals[i+1] for i in range(len(vals)-1)) or all(vals[i]>vals[i+1] for i in range(len(vals)-1))) and all(0 < abs(vals[i]-vals[i+1]) < 4 for i in range(len(vals)-1))
+
+
+def question_1_2(filename, skip_allowed):
```

Comment:
Suggest renaming this to something like `get_solution` or `count_safe_reports_from_file`